themes/scss/widgets/default.scss

```diff
@@ -26,12 +39,17 @@
   margin-right: $sMargin-section;
   margin-left: $sMargin-section;
 }
+.Widget-header--timeSeries {
```

Comment:
After looking into it, I don't see that it's feasible. If you look [further down in the same file](https://github.com/CartoDB/cartodb.js/pull/879/files#diff-761c5e7aef3c026da769f41f9e4447dbL49) I have other overrides of the layout that are specific for this context, so I need this modifier either way.